deltachat-ios/Controller/ChatListController.swift

```diff
@@ -251,10 +251,10 @@ class ChatListController: UITableViewController {
     @objc func applicationDidBecomeActive(_ notification: NSNotification) {
         if navigationController?.visibleViewController == self {
             if !isInitial {
-                isInitial = false
                 startTimer()
-                refreshInBg()
             }
+            handleChatListUpdate()
```

Comment:
i think, the `handleChatListUpdate()` can go back to the `!isInitial` condition - if it is the initial load, things are created correctly anyway, calling `handleChatListUpdate()` explicitly in this case would result in one unneeded update.

Reply:
🤦 right